examples/pi-reduce_vs_atomic.cpp

```diff
@@ -140,20 +143,21 @@ int main(int RAJA_UNUSED_ARG(argc), char** RAJA_UNUSED_ARG(argv[]))
 
   *atomic_pi = 0.0;
 
+  // clang-format off
```

Comment:
Why did the script insert the clangt-format off comment here, but not at line 129 above?